addons/website_links/static/src/js/website_links_code_editor.js

```diff
@@ -44,35 +44,46 @@ publicWidget.registry.websiteLinksCodeEditor = publicWidget.Widget.extend({
      * @param {String} newCode
      */
     _showNewCode: function (newCode) {
-        $('.o_website_links_code_error').html('');
-        $('.o_website_links_code_error').hide();
+        // Clear error message and hide error element
+        const errorElement = this.el.querySelector(".o_website_links_code_error");
+        errorElement.innerHTML = "";
+        errorElement.style.display = "none";
 
-        $('#o_website_links_code form').remove();
+        // Remove existing form element
+        this.el.querySelector("#o_website_links_code form")?.remove();
 
         // Show new code
-        var host = $('#short-url-host').html();
-        $('#o_website_links_code').html(newCode);
-
-        // Update button copy to clipboard
-        $('.copy-to-clipboard').attr('data-clipboard-text', host + newCode);
-
-        // Show action again
-        $('.o_website_links_edit_code').show();
-        $('.copy-to-clipboard').show();
-        $('.o_website_links_edit_tools').hide();
+        const host = this.el.querySelector("#short-url-host").innerHTML;
+        const codeElement = this.el.querySelector("#o_website_links_code");
+        if (codeElement) {
+            codeElement.innerHTML = newCode;
+
+            // Update button copy to clipboard
+            this.el
+                .querySelector(".copy-to-clipboard")
+                ?.setAttribute("data-clipboard-text", host + newCode);
+
+            // Show action again
+            this.el.querySelector(".o_website_links_edit_code").style.display = "";
+            this.el.querySelector(".copy-to-clipboard").style.display = "";
+            this.el.querySelector(".o_website_links_edit_tools").style.display = "none";
```

Comment:
use class ```d-none``` by adding/removing class.

Check all places where we did such changes.